misc/scripts/install-local.sh

```diff
@@ -100,12 +100,12 @@ if [[ -n "$build_depends" ]]; then
     fi
 
 if [[ -n "$pacdeps" ]]; then
-    declare -a pacdepslist
     for i in "${pacdeps[@]}"
     do
         fancy_message info "Installing $i"
+        echo "PACSTALL_DEPENDS=true" | sudo tee /tmp/pacstall-pacdeps
```

Comment:
```suggestion
        sudo touch /tmp/pacstall-pacdeps
```
There is no need of echoing anything there, I think, because in the detector, you are just checking if the file exists